### PR TITLE
Bridge

### PR DESCRIPTION
**Pull request by drewstone**

### CODE REVIEW

File: src/bridged_mt/mod.rs [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+			&Scalar::from(set[i]),
+		);
+
+		let (_, _, o) =
+			cs.multiply(product.clone(), diff_vars[i].variable.into());
+		product = o.into();
+	}
+
+	// Ensure product of elements if `diff_vars` is 0
+	cs.constrain(product);
```
Comment (lazovicff) [inline src/bridged_mt/mod.rs:55]: Nice solution

**Review by lazovicff — APPROVED** [review]
OK to merge after errors are fixed

**Review by shekohex — APPROVED** [review]
LGTM!

**Comment by drewstone** [comment]
Yea these errors are weird

**Pull request merged**